AndroidSDKCore/src/main/java/com/leanplum/EventsUploadInterval.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020, Leanplum, Inc. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.leanplum;
+
+/**
+ * The enumeration represents time interval to periodically upload events to server.
+ * Possible values are 5, 10, or 15 minutes.
+ */
+public enum EventsUploadInterval {
```

Comment:
What do you think about:
`PeriodicSyncInterval`

Reply:
It is only about uploading data to server. As I understand what 'sync' means is that the data on device will be synced with data on server, but that will happen only on Start request, if you don't have the Start event in the DB it won't download data from server.

Reply:
Yeah actually we talked about that, I was thinking about `PeriodicUploadInterval` but now when I wrote it probably `EventsUploadInterval` sounds better